### ISSUE TITLE
Is it possible to run parsec and zerotier app?

### ISSUE THREAD

**Issue by kevinshane** [Thinstation/thinstation#653]
Hi dear TSer, I have some questions for thinstation build:
1. Is it possible to compile/run parsec? Link here: `https://parsec.app/` They do provide a linux deb for ubuntu based distro. But I just don't know how to compile it and run inside a TS build.
2. Is it possible to compile/run zerotier app? Link here: `https://www.zerotier.com/` or a similar app like `TileSacle` as well. The reason why we need this, is because we often remote into office pc at home. `Zerotier` is stable and handy.
Thanks in adv

**Comment by Thinstation** [comment]
Yes. You can use the ubuntu .deb packages, there is just no apt or dpkg to do the installation for you. You have to manually decompress the archives, place the files into a package and likely handle a few fixups to make everything work, but it is doable.

**Comment by Thinstation** [comment]
You can look at the chome, teams, talkplugin and edge packages for examples of using .deb in ThinStation
